typescript/packages/common-memory/test/consumer-test.ts

```diff
@@ -66,7 +73,7 @@ test("query empty memory", store, async (session) => {
 });
 
 test("create new memory", store, async (session) => {
-  const consumer = Consumer.open({ as: alice, session });
+  const consumer = Consumer.open({ as: alice, session, clock: new Clock() });
```

Comment:
I think you need to create one clock and pass it in both places or you may still end up with intermitten failures